tests/sentry/api/endpoints/test_release_deploys.py

```diff
@@ -15,7 +17,8 @@ def test_simple(self):
         )
         release = Release.objects.create(
             organization_id=project.organization_id,
-            version='1',
+            # test unicode
+            version='1–0',
```

Comment:
fwiw, instead of using the `# -*- coding` thing, you could use the escaped format here: `u'1\u20130'`.